examples/django_example/example/settings.py

```diff
@@ -201,7 +201,9 @@
     'social.backends.vend.VendOAuth2',
     'social.backends.email.EmailAuth',
     'social.backends.username.UsernameAuth',
+    'social.backends.trinity.TrinityOauth2',
```

Comment:
We probably don't need to include this here unless we're pushing this upstream. Our use case seems a bit too specific for that.